### DISCUSSION TITLE
How can AutoUpdater be localized?

### DISCUSSION THREAD

**Discussion by roscofil** [ravibpatel/AutoUpdater.NET#714] [General]
Hello,
I happened to use AutoUpdater once a while back (about 2 years ago) and I recall that, at that time, there used to be a 
`AutoUpdater.CurrentCulture = CultureInfo.CreateSpecificCulture("xx"); ` method where xx could be anything like en, fr, de etc...
Did this method disappear now or how can I localize the output messages from AutoUpdater such as "No update for now, please try later" or so?
Thx again for an excellent tool!
Cheers

**Comment by ravibpatel** [comment]
You can do that as shown [here](https://github.com/ravibpatel/AutoUpdater.NET/blob/master/AutoUpdaterTest/MainWindow.xaml.cs#L19-L21).

**Comment by roscofil** [comment]
Hello,

Thanks for your reply.

Not sure where I have to change anything here…French OS, French thread, so where should the CultureInfo be changed?

Sorry but not really familiar with this…

Thx

Philippe

Le 28 sept. 2024 à 09:36, Ravi Patel ***@***.***> a écrit :

﻿

You can do that as shown here<https://github.com/ravibpatel/AutoUpdater.NET/blob/master/AutoUpdaterTest/MainWindow.xaml.cs#L19-L21>.

**Comment by ravibpatel** [comment]
It should show the dialog in French by default if your default language on the OS is French. Are you still getting English dialogs?

**Comment by roscofil** [comment]
Yes, I do. 
I use a forced update and so, when no update is needed, I still get the « No update found » info box in English. 
using latest 1.9.2 version

**Comment by roscofil** [comment]
BTW, saw somebody from Germany seeming to have the same issue with his EXE in Issues…

**Comment by ravibpatel** [comment]
Do you have satellite assemblies for all languages in the folder where your executable resides?

**Reply by roscofil** [reply]
Yes they are all under a FR subdir. 
Can’t check now if the subdir is in the exe path. Will check when back.

**Comment by roscofil** [comment]
So, this is a non-issue...
You were right : I did not package the fr subdir into my installer, which is why all this happened.
Sorry for not realizing this earlier :-(
Thank you again for your help on this!
Philippe
